database-postgresql/helpers.js

```diff
@@ -0,0 +1,62 @@
+
+const client = require('./connection');
+
+const getPhotos = (trailId, sortOrder, callback) => {
+  let sortOrderStatement = sortOrder ? `ORDER BY upload_date ${sortOrder}` : '';
```

Comment:
What if sortOrder is truthy but not a string? Or is a string but invalid? "derp" or "{"sortOrder": "asc"}

Reply:
fixed - thanks!